src/pages/plugins/tutorials/importing-modules/index.md

```diff
@@ -0,0 +1,37 @@
+# JavaScript Modules
+
+The beauty of plugins is that it allows you to have more than just one JavaScript file. And in order to manage elaborate projects, you might want to modularize your code and manage it from separate JS files.
```

Comment:
@kareemmyk  Can you please validate that this works?

Reply:
The details are correct.

However, we should mention a few items.
For example if the starting point of a plugin is a ```JavaScript``` file then it must be web packed.
WebPack is not needed if if an ```index.html``` is the starting point for a plugin and it  invokes multiple ```js``` files.

Reply:
Good to know! I have noted this point. And will add it to the plugins -> tutorial -> adding-command-entrypoint details.